docs/presto-admin-commands.rst

```diff
@@ -392,9 +392,17 @@ server install
 **************
 ::
 
-    presto-admin server install <local_path> [--nodeps]
+    presto-admin server install <rpm_specifier> [--rpm-source] [--nodeps]
 
-This command copies the presto-server rpm from ``local_path`` to all the nodes in the cluster, installs it, deploys the general presto configuration along with tpch connector configuration. The ``local_path`` should be accessible by ``presto-admin``.
+This command takes in a parameter ``rpm_specifier``, which must be a string describing the presto-server rpm to be used.
+The string can be a local path to a previously downloaded rpm. The local path should be accessible by ``presto-admin``.
+The string can be a version number. In this case, this command will use the argument ``--rpm-source`` to determine where to download the rpm from. This command attempts to find a local rpm that matches the rpm that would be downloaded. If no match is found, it will attempt to download the rpm.
```

Comment:
you should also note that the default value for --rpm-source is 'facebook' since it's an optional argument.